services/todo-legacy/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es6",
```

Comment:
I'd personally recommend `es2017` together with a note in README saying that this app requires Node.js 8.x. and `engines` field in `package.json` set accordingly.